storage/core/src/main/java/com/hortonworks/registries/storage/transaction/TransactionEventListener.java

```diff
@@ -90,18 +90,12 @@ public void onEvent(RequestEvent event) {
                     isTransactionActive = true;
                 }
             } else if (eventType == RequestEvent.Type.RESP_FILTERS_START) {
-                // not supporting transactions to filters
```

Comment:
We may soon forget why this change was done. :) 
You can put a comment on why this change is done.

Reply:
@satishd  Done